src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -204,7 +204,7 @@ public enum ErrorCodeV1 {
   TABLE_MISSING_PARTITIONING_KEYS("Key needs to be provided as part of partitioning definition"),
   TABLE_COLUMN_DEFINITION_MISSING("Column definition is missing for the provided key field"),
   TABLE_COLUMN_TYPE_UNSUPPORTED("Unsupported column types"),
-  TABLE_COLUMN_UNKNOWN("Column unknown");
+  TABLE_INVALID_FILTER("Invalid table filter");
```

Comment:
-1 on added a new V1 error code, can we remove this pls

Reply:
Thanks, removed and change to use v2 ERROR_CODE INVALID_FILTER.